api/src/main/java/org/pmiops/workbench/profile/ProfileMapper.java

```diff
@@ -78,4 +84,16 @@ default List<String> splitAccessTierShortNames(
           .collect(Collectors.toList());
     }
   }
+
+  default boolean checkInitialCreditsExtensionEligibility(DbUser dbUser) {
```

Comment:
Seemed easier to calculate extension on the back end then having to calculate it in several places in the UI.